ui/src/literature/containers/DetailPageContainer/DetailPageContainer.jsx

```diff
@@ -130,6 +131,13 @@ function DetailPage({
                         pidValue={controlNumber}
                       />
                     )}
+                    {datasetLinks &&
+                    <UrlsAction
+                    urls={datasetLinks}
+                    icon={<DatabaseOutlined />}
+                    text="datasets"
+                        />
+                        }
```

Comment:
I believe you will have to update tests and e2e tests

Reply:
They pass, but I can update the snapshot in e2e

Reply:
It's important to add the link to the record, so we make sure it displayed correctly :)

Reply:
We could add the link to test the snapshot :)